index.js

```diff
@@ -59,3 +77,5 @@ module.exports = function requireAll(options) {
 function identity(val) {
   return val;
 }
+
+function isObject(arg) { return Object.prototype.toString.call(arg).indexOf('Object') !== -1; }
```

Comment:
now I remember I use this because in my projects I am trying to distinguish between a "real" object and an array.  So probably need as one could export just an array and it would need a prop/key.  Otherwise it would throw an error.  I'll add a new test with both and see if it fails on the "usual"